force-app/main/default/lwc/builderSearchResults/builderSearchResults.js-meta.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+	<isExposed>true</isExposed>
+	<targets>
+		<target>lightningCommunity__Page</target>
+		<target>lightningCommunity__Default</target>
+	</targets>
```

Comment:
remove extra space

Reply:
all these look like they have an extra tab, they should be aligned with the `<masterLabel>`, same for the `<targetConfigs>`